crates/red_knot_python_semantic/resources/mdtest/attributes.md

```diff
@@ -169,6 +169,10 @@ class C:
     pure_class_variable1: ClassVar[str] = "value in class body"
     pure_class_variable2: ClassVar = 1
 
+    def method(self):
+        # TODO: this should be an error
+        self.pure_class_variable1 = "value set through instance"
```

Comment:
We can't do this yet, but should be able to, as soon as we know what the type of `self` is.